paper.tex

```diff
@@ -102,7 +102,16 @@ \section{Introduction}
 
 
 \section{Related Work}
+The publish/subscribe service model consists of publishers, subscribers and brokers, which has been proved\cite{pubSubAnalysis, pubSubAnalysis2} to be an efficient and flexible solution for a large number of diverse entities like IoT applications. 
```

Comment:
While referring prior art, we shall emphasize on the following aspects:
1. generic encrypted data communication;
2. privacy-enhanced data subscriptions;
3. the rights for accessing digital materials;
4. storage itself is indeed a vital considerations for IoT and mobile computing;
5. incentive for data economics;